docroot/themes/humsci/humsci_basic/patterns/grid/grid.html.twig

```diff
@@ -0,0 +1,16 @@
+{%
+  set classes = [
+  'hb-grid',
+  'hb-grid--cols-' ~ columns,
+]
+%}
+
+{% if items|length > 0 %}
+<div{{ create_attribute({'class': classes}) }}>
```

Comment:
`attributes` should be an available variable in each twig template. This should just need `attributes.addClass(classes)`